### PR TITLE
features/CIV-5478-proceed against full dispute

### PR DESCRIPTION
**Pull request by laperla11**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-5478

### Change description ###
Develop the "Intention to Proceed" screen in the scenario where the Defendant has fully disputed the story

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by laarmada — APPROVED** [review]

**Review by neeta-hmcts — APPROVED** [review]

**Review by portizdev — COMMENTED** [review]
There is a typo error on the folder name. It's claimant-reponse and should be claimant-response

File: src/main/routes/urls.ts [hunk 1]
```diff
@@ -177,3 +178,5 @@ export const CLAIMANT_PHONE_NUMBER_URL = `${BASE_CLAIM_URL}/claimant-phone`;
 export const CLAIMANT_SOLE_TRADER_DETAILS_URL = `${BASE_CLAIM_URL}/claimant-sole-trader-details`;
 export const CLAIMANT_INTEREST_RATE_URL = `${BASE_CLAIM_URL}/interest-rate`;
 export const CLAIMANT_INTEREST_DATE_URL = `${BASE_CLAIM_URL}/interest-date`;
+export const CLAIMANT_RESPONSE_INTETNTION_TO_PROCEED_URL = `${BASE_CLAIMANT_RESPONSE_URL}/intention-to-proceed`;
```
Comment (portizdev) [inline src/main/routes/urls.ts:181]: Typo error: INTETNTION

File: src/main/routes/features/claimant-reponse/claimantIntentionToProceedController.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    renderView(new GenericForm(claimantResponse.intentionToProceed), res);
+  } catch (error) {
+    next(error);
+  }
+});
+
+claimantIntentionToProceedController.post(CLAIMANT_RESPONSE_INTETNTION_TO_PROCEED_URL, async (req, res, next) => {
+  try {
+    const claimId = req.params.id;
+    const claimantIntentionToProceed = new GenericForm(new GenericYesNo(req.body.option, 'ERRORS.VALID_YES_NO_SELECTION'));
```
Comment (kenneth-hmcts) [inline src/main/routes/features/claimant-reponse/claimantIntentionToProceedController.ts:35]: I am trying to make a common service for the genericYesNo in ClaimantResponseService in my ticket. Is it possible to use that?

File: src/main/routes/urls.ts (hunk 1 above)
Comment (laperla11) [reply src/main/routes/urls.ts]: done

File: src/main/routes/features/claimant-reponse/claimantIntentionToProceedController.ts (hunk 2 above)
Comment (laperla11) [reply src/main/routes/features/claimant-reponse/claimantIntentionToProceedController.ts]: we already have a generic service as `claimantResponseService` and I used it,
we can discuss your approach, if everyone agrees, I can use that one as well

**Comment by laperla11** [comment]
created a new PR for pipeline issues

**Pull request closed without merging**